openverse_catalog/dags/common/slack.py

```diff
@@ -1,5 +1,5 @@
 """
-# Slack Block Message Builder
+# Slack Block Message Builder.
```

Comment:
This is yet another silly change to be forced to make :cry:

Reply:
It might be a difference in the style of documentation but the API docs mostly use sentences and so this comment there would've been "This is a message builder for Slack.", with the full-stop being there by default due to the sentence-like nature of the comments.